projects/storefrontlib/src/lib/user/user.module.ts

```diff
@@ -6,8 +6,13 @@ import { ReactiveFormsModule } from '@angular/forms';
 
 import { LoginModule } from './login/login.module';
 import { RegisterComponent } from './register/register.component';
-import { ResetPasswordComponent } from './reset-password/reset-password.component';
-import { UserModule, UrlTranslationModule } from '@spartacus/core';
+import { ForgotPasswordComponent } from './forgot-password/forgot-password.component';
```

Comment:
Can we create "ForgotPasswordModule"  and in UserModule, we can just import it?

Reply:
I agree.  This would be a better and more modular design.